docs/.vuepress/styles/index.styl

```diff
@@ -39,10 +39,34 @@ pre.vue-container
     0 1px 10px 0 rgba(0, 0, 0, 0.12);
 }
 
+.padding-1 {
+  padding: 1.2em;
+}
+
 .primaryBtn {
   background-color: #037dd6;
 }
 
 .greenBtn {
   background-color: #4caf50;
 }
+
+.navbar, .links {
+  background-color: #D7C1B3 !important
```

Comment:
Wow, bold choice in colour :sweat_smile: Seems a bit...plain, and off-brand for us maybe? :man_shrugging: Maybe it's just me.

Reply:
I got it from ya'lls logo and I think design palette. Could someone from ya'lls design team confirm please? 🙂

Reply:
This is the one thing I don't like! I'd prefer that we just go with white unless we find something more suitable. Maybe @cjeria has some ideas.

Reply:
Design agrees that white header is better for now :)